lib/assets/javascripts/manage-prototype/manage-plugins.js

```diff
@@ -0,0 +1,110 @@
+window.GOVUKPrototypeKit.documentReady(() => {
+  const params = new URLSearchParams(window.location.search)
+  const packageName = params.get('package')
+  const mode = params.get('mode') || window.location.pathname.split('/').pop()
+  let startTimestamp = ''
+  let requestTimeoutId
+  let timedOut = false
+  const timeout = 30 * 1000
+  const token = document.querySelector('meta[name="csrf-token"]').getAttribute('content')
+
+  const show = (id) => document.getElementById(id)
+    .classList.remove('govuk-!-display-none')
+
+  const hide = (id) => document.getElementById(id)
+    .classList.add('govuk-!-display-none')
```

Comment:
I think it would be preferable to use the `hidden` attribute instead of a CSS class, that way the page is more likely to look correct if the stylesheets haven't loaded or have been overridden for some reason.

Currently without stylesheets the page to install a plugin will read:

```
Install Step By Step from GOV.UK Prototype Kit

In terminal, press ctrl + c to stop your prototype, then run:

npm install @govuk-prototype-kit/step-by-step

When you've installed the plugin, restart your prototype in the terminal by typing:

npm run dev

Install
Installing ...
Install complete
There was a problem installing
[Please contact support](https://prototype-kit.service.gov.uk/docs/support)
```
<img width="948" src="https://user-images.githubusercontent.com/503614/207647301-4c6ed231-19b2-4547-b83d-f729eaa37f28.png">

Reply:
Ideally this page would work without CSS (in fact it does, if you click the button! You just can't tell from the browser...)

Reply:
Perhaps if they are in this situation they have more issues than what is wrong with the page.  The last comment to cal support is probably a good call

Reply:
The hidden attribute works when there is no css however it does not hide the button so I have implemented both css and the attribute to hide and show.

Reply:
Fixed using the hidden attribute and some small markup changes